src/test/java/com/google/devtools/build/lib/runtime/commands/CleanCommandRecommendsAsyncTest.java

```diff
@@ -55,6 +58,13 @@ public CleanCommandRecommendsAsyncTest(List<String> commandLine, OS os, boolean
     this.expectSuggestion = expectSuggestion;
   }
 
+  @After
+  public void stopProfilers() throws Exception {
```

Comment:
Actually, now that I look closer, this test is not simulating crashes. It just happens to be failing with a crash, which then poisons subsequent test cases. I suspect that if you fix or comment out the already failing case, then the other cases past.

A better solution to prevent this might be to move https://github.com/bazelbuild/bazel/blob/e6751ed25615e0ce6e5ecee68dd6010b2de2225c/src/main/java/com/google/devtools/build/lib/runtime/BlazeRuntime.java#L680-L685 to https://github.com/bazelbuild/bazel/blob/e6751ed25615e0ce6e5ecee68dd6010b2de2225c/src/main/java/com/google/devtools/build/lib/runtime/BlazeCommandDispatcher.java#L624 (also then need to add it to BlazeRuntimeWrapper).